### ISSUE TITLE
Error after OOM

### ISSUE THREAD

**Issue by turbotankist** [VictoriaMetrics/VictoriaMetrics#148]
**Describe the bug**
We are run victoriametrics in kubernetes pod with some memory limit.
At one time it was killed by OOM but after restarting there can run again.
```
2019-08-12T10:26:30.512+0000	fatal	app/vmstorage/main.go:47	cannot open a storage at /storage with retention period 6 months: cannot open table at "/storage/data": cannot open partitions in the table "/storage/data": cannot open partition "2019_08": cannot open small parts from "/storage/data/small/2019_08": cannot run transactions from "/storage/data/small/2019_08": cannot run transaction from "/storage/data/small/2019_08/txn/15B9E32E3896E8E0": invalid last line in the transaction file: got ""; must contain `srcPath -> dstPath`
```
**Expected behavior**
Expected normal run after restart.

**Version**
```
image: victoriametrics/victoria-metrics:v1.22.2
```

**Comment by valyala** [comment]
Thanks for the bug report.

VictoriaMetrics continuously merges small parts with data into bigger parts.  It creates `transaction` files with instructions on how to atomically substitute smaller parts by bigger part after the merge is complete. It turns out that the `transaction` file can be corrupted on unclean shutdown like in your case. The issue will be fixed in the next release. Temporary workaround - to remove the broken `transaction` file at `/storage/data/small/2019_08/txn/15B9E32E3896E8E0` and start VictoriaMetrics again.

**Comment by valyala** [comment]
The issue should be fixed in 5d8d11001054acb93979ebf2224dedec8e13657f .

**Comment by valyala** [comment]
Closing the issue, since it must be fixed in [v1.25.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases).
